docs/governance.rst

```diff
@@ -0,0 +1,129 @@
+Binder Project Governance
```

Comment:
Maybe the file should be called `governance_binder.rst`? I don't think that jupyterhub has its own governance structure, but this might help disambiguate things a bit